opaque_keys/edx/tests/test_locators.py

```diff
@@ -59,3 +61,59 @@ def test_version_tree(self):
         test_id = ObjectId(test_id_loc)
         valid_locator = CourseLocator(version_guid=test_id)
         self.assertEqual(VersionTree(valid_locator).children, [])
+
+
+@ddt.ddt
+class CourseLocatorV2Tests(TestCase):
+    """
+    Tests for :class:`.CourseLocatorV2`
+    """
+
+    def test_course_locator_v2(self):
+        """
+        Verify that the method "from_string" of class "CourseKeyV2"
+        returns an object of "CourseLocatorV2" for a valid course key.
+        """
+        course_key_v2 = 'course-v2:org+course'
+        course_locator_v2 = CourseKeyV2.from_string(course_key_v2)
+        expected_course_locator = CourseLocatorV2(org='org', course='course')
+        self.assertEqual(expected_course_locator, course_locator_v2)
+
+    @ddt.data(
+        'org/course/run',
+        'course-v1:org+course+run',
+    )
+    def test_course_locator_v2_from_course_key(self, course_id):
```

Comment:
You don't need to prefix your test methods with `course_locator_v2`. The fact that `CourseLocatorV2` is given by the name of the test class, `CourseLocatorV2Tests`. This should simply be `test_from_course_key`.